text/0024-track-validator-set-when-synchronizing.org

```diff
@@ -0,0 +1,141 @@
+#+STARTUP: inlineimages
+
+* Joining Node Validator Set Tracking When Synchronizing
+:PROPERTIES:
+:CUSTOM_ID: joining-node-validator-set-tracking-when-synchronizing
+:END:
+
+** Summary
+:PROPERTIES:
+:CUSTOM_ID: summary
+:END:
+
+CEP PR: [[https://github.com/casperlabs/ceps/pull/0024][casperlabs/ceps#0024]]
+
+A node joining the network needs a secure means of tracking the
+validator set as it processes blocks. To achieve this, the validators
+for the next era are to be provided in each switch block.
+
+This is required for [[https://github.com/casperlabs/ceps/pull/1234][casperlabs/ceps#1234]] and [[https://github.com/casperlabs/ceps/pull/9999][casperlabs/ceps#9999]].
```

Comment:
These are placeholders for some future CEPs, right? I would make it clearer, for example like this:

"This is required for [Feature 1 (placeholder link)](https://github.com/casperlabs/ceps/pull/1234) and [Feature 2 (placeholder link)](https://github.com/casperlabs/ceps/pull/9999)."

Placeholders below could also be replaced with links like that.

Reply:
Fixed in https://github.com/CasperLabs/ceps/pull/24/commits/351be7f14f7c9a6287a4d3f2e86e8e9f30840e9d

Reply:
Got rid of these in https://github.com/CasperLabs/ceps/pull/24/commits/17e1433250bcc8b81edd0ecbf53ed8679821e023